build-system/babel-plugins/babel-plugin-transform-stringish-literals/index.js

```diff
@@ -14,16 +14,12 @@
  * limitations under the License.
  */
 
-const MERGEABLE_TYPES = ['StringLiteral', 'NumericLiteral', 'TemplateLiteral'];
+const TILDA_REGEXP = /\x60/g;
 
 module.exports = function ({types: t}) {
   const cloneNodes = (nodes) => nodes.map((node) => t.cloneNode(node));
-  const escapeStringForTemplateLiteral = (value) =>
-    String(value).replace(/\x60/g, '\\`');
-  const canMergeBinaryExpression = (binaryExpression) =>
-    MERGEABLE_TYPES.includes(binaryExpression.left.type) &&
-    MERGEABLE_TYPES.includes(binaryExpression.right.type) &&
-    binaryExpression.operator === '+';
+  const escapeForLiteral = (value) =>
+    String(value).replace(TILDA_REGEXP, '\\`');
```

Comment:
We also need to escape `${}` blocks: https://github.com/ampproject/amphtml/blob/master/build-system/babel-plugins/babel-plugin-transform-json-configuration/index.js#L163

Reply:
Should be resolved with 95f48f1.